pkg/operator/ceph/csi/template/rbd/csi-rbdplugin.yaml

```diff
@@ -106,6 +106,11 @@ spec:
               mountPath: /tmp/csi/keys
             - name: host-run-mount
               mountPath: /run/mount
+            {{ if .EnablePluginSelinuxHostMount }}
```

Comment:
@humblec https://github.com/ceph/ceph-csi/issues/2295 is reported only for cephfs. Do we need changes in rbd alss?

Reply:
Enabling selinux relabelling is equally applicable for rbd too. so I have added it for rbd also.  @Madhu-1